### PR TITLE
fixed typeorm example

### PR DESCRIPTION
**Pull request by david-piiano**
Moved to a new Vault
Moved to yarn

### CODE REVIEW

File: yarn.lock
Comment (ori-shalom) [inline yarn.lock:1]: remove

File: sdk/yarn.lock
Comment (ori-shalom) [inline sdk/yarn.lock:1]: remove

File: examples/typeorm-encryption/run-vault.sh [hunk 1]
```diff
@@ -55,10 +59,8 @@ user PERSONS (
   ssn SSN COMMENT 'Social security number',
 )"
 
-# Install dependencies
-npm ci
+# Build
+yarn build
```
Comment (ori-shalom) [inline examples/typeorm-encryption/run-vault.sh:63]: Revert - The `start` script is using `ts-node` to run ts files directly so need to run `yarn build` but we do need to install dependencies so `npm ci` is needed.
```suggestion
# Install dependencies
npm ci
```

Comment (david-piiano) [reply examples/typeorm-encryption/run-vault.sh:63]: it did work for me. The CI actually didn't but perhaps due to another problem

**Review by ori-shalom — APPROVED** [review]

**Pull request merged**